cmd/standalone/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.7
+EXPOSE 5775 6831 6832 14267 3000 3001 8080 8081 8082 8083
+
+ADD . /go/src/github.com/uber/jaeger
+
+RUN apt-get update
+RUN apt-get install -y curl
+RUN curl -sL https://deb.nodesource.com/setup_6.x | bash
+RUN apt-get install -y nodejs
+RUN node -v
+RUN npm -v
+
+RUN git clone https://github.com/uber/jaeger-ui
```

Comment:
My preference is to use a git submodule. Publishing to NPM is unnecessary overhead & release friction.